src/js/plugin/sdk/pipeline/PipelineStore.js

```diff
@@ -0,0 +1,21 @@
+import PipelineModel from "./PipelineModel";
+
+export default class PipelineStore {
```

Comment:
We should export and instance of `PipelineStore` as the pipeline store will create and manage the `PipelineModel` should be a _singleton_ similar to all of our stores.